judgments/views/advanced_search.py

```diff
@@ -70,7 +70,7 @@ def advanced_search(request):
         query_without_stop_words = preprocess_query(query_params["query"])
         search_parameters = SearchParameters(
             query=query_without_stop_words,
-            court=query_params["court"],
+            court=",".join(query_params["court"]),
```

Comment:
This stems from: 
https://github.com/nationalarchives/ds-caselaw-custom-api-client/blob/main/src/caselawclient/search_parameters.py#L12

where `SearchParameters` expects court to be of type `str` or `None`

This is currently misleading, as there is still the legacy code on https://github.com/nationalarchives/ds-caselaw-custom-api-client/blob/main/src/caselawclient/search_parameters.py#L63

which means that the api client search can handle court as a list or a string.

This view was the only place where it was actually utilising the ability to pass in a list, but in https://github.com/nationalarchives/ds-caselaw-public-ui/pull/738 I changed it to be a string since mypy was complaining about the above.

Hadn't realised that the query_params object was being used in the view through the context!

This change fixes the bug and complies with search_params.

We should probably revist this, and get rid of the apiclients ability to accept list or string since that is confusing, and then come back to PUI and make sure this is still working as expected.